flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/output/SnapshotUtilsTest.java

```diff
@@ -50,13 +53,24 @@
     private static final List<String> ACTUAL_ORDER_TRACKING =
             Collections.synchronizedList(new ArrayList<>(EXPECTED_CALL_OPERATOR_SNAPSHOT.size()));
 
+    private static SnapshotType actualSnapshotType;
+
     @Test
     public void testSnapshotUtilsLifecycle() throws Exception {
+        testSnapshotUtilsLifecycleWithSavepointFormatType(SavepointFormatType.DEFAULT);
+        testSnapshotUtilsLifecycleWithSavepointFormatType(SavepointFormatType.NATIVE);
```

Comment:
In general, we should only use one unit test case for one specific test purpose. It's better to split this test case into two cases, one is testing `SavepointFormatType.DEFAULT` and another is for `SavepointFormatType.NATIVE`.

Moreover, I also suggest to test `SavepointFormatType.CANONICAL` instead of `SavepointFormatType.DEFAULT` in case we might change the `SavepointFormatType.DEFAULT` to `SavepointFormatType.NATIVE` in the future.

Reply:
THX for your remind, actually I have already tested the generated savepoint locally and I tested it again just. The result met my request perfectly.
And I split the test case in the new commit, could you help take a glance?